src/WorkItemMigrator/JiraExport/JiraProvider.cs

```diff
@@ -435,19 +461,18 @@ public string GetCustomId(string propertyName)
                 .ToLookup(l => l.key, l => l.value);
         }
 
```

Comment:
When searching for fields that match the given propertyName, just return them all, rather than picking the first one.